FLIR/conservator/wrappers/image.py

```diff
@@ -1,8 +1,25 @@
 from FLIR.conservator.generated import schema
-from FLIR.conservator.wrappers.media import MediaType
+from FLIR.conservator.util import md5sum_file
+from FLIR.conservator.wrappers.media import MediaType, MediaCompare
+from FLIR.conservator.wrappers.type_proxy import requires_fields
 
 
 class Image(MediaType):
     underlying_type = schema.Image
     by_id_query = schema.Query.image
     search_query = schema.Query.images
+
+    @requires_fields("image_md5")
+    def compare(self, local_path):
+        """
+        Use md5 checksums to compare image contents to local file
+
+        Returns result as MediaCompare object
+
+        :param local_path: Path to local copy of file for comparison with remote.
+        """
+        result = MediaCompare.MISMATCH
+        local_md5 = md5sum_file(local_path)
+        if local_md5 == self.image_md5:
+            result = MediaCompare.MATCH
+        return result
```

Comment:
Very similar code in `Video`--can we extract to helper such as `MediaCompare.with_local_file(local_path, remote_md5)`?

```suggestion
        local_md5 = md5sum_file(local_path)
        if local_md5 == self.image_md5:
            return MediaCompare.MATCH
        return MediaCompare.MISMATCH
```

Reply:
Comparing a couple checksums isn't inherently a media-specific task; we might want to run validation on file-locker or metadata files later on. How about moving more into util, including a generically named enum "FileCompare".

Reply:
On the other hand Conservator isn't providing md5sum for file-locker or metadata files yet, so maybe it is media-specific

Reply:
Either way is fine with me

Reply:
I went with your original suggestion of media for above reason